texts/index.js

```diff
@@ -227,3 +227,41 @@ texts.push([
 	"Logo bởi", // 32
 	"Chọn ngôn ngữ" // 33
 ])
+
+// Dutch (Translation provided by jelle619)
+texts.push([
+	"Over", // 0
+	"Download", // 1
+	"Gemeenschap", // 2
+	"Compatibiliteit", // 3
+	"Showcase", // 4
+	"Draag bij", // 5
+	"Doneer", // 6
+	"’s Werelds eerste PlayStation Vita emulator.", // 7
+	"Kom Meer Te Weten", // 8
+	"Over Vita3K", // 9
+	"Vita3K is ‘s werelds eerste experimentele open-source PlayStation Vita emulator voor Windows en MacOS.", // 10
+	"Wees ervan bewust dat het in staat stellen van illegale activiteiten niet de bedoeling van deze emulator is.", // 11
+	"Kies een passende versie!", // 12
+	"Binnenkort!", // 13
+	"Haal binnen via", // 14
+	"Neem contact met ons op!", // 15
+	"Neem contact op met de ontwikkelaars en andere gebruikers wanneer je maar wilt. Doe dit het liefst via Discord, maar de keuze ligt bij jou!", // 16
+	"Doe mee!", // 17
+	"Kijk wat we aan het doen zijn!", // 18
+	"De emulator kan nog geen <b>commerciële spellen</b> afspelen. <br>Er worden een paar homebrew-spellen ondersteund. Zie de <a href=\"compatibility.html?lang=en\">compatibiliteitslijst</a> voor verdere informatie!", // 19
+	"Homebrew Game Port", // 20
+	"Homebrew Game", // 21
+	"Homebrew Hulpmiddel", // 22
+	"Help ons beter worden!", // 23
+	"Je kunt op de volgende manieren bijdragen aan de emulator:", // 24
+	"Licht ons in over problemen of maak Pull Requests op onze <a href=\"https://github.com/Vita3K/Vita3K\">GitHub repository</a>.", // 25
+	"Update de <a href=\"https://github.com/Vita3K/homebrew-compatibility/issues\">homebrew compatibiliteitslijst</a>.", // 26
```

Comment:
compatibiliteitslijst?

Reply:
Yup.
One word.
No spaces.

Reply:
lol

Reply:
Well okay technically they’re still two words but it has to be written without a space inbetween.